presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -180,6 +191,10 @@ Accessors
     Returns the 2-dimensional cartesian minimum distance (based on spatial ref)
     between two geometries in projected units.
 
+.. function:: ST_SphericalDistance(SphericalGeometry, SphericalGeometry) -> double
```

Comment:
Let's keep the `ST_Distance` name and move this function next to original one.

Reply:
@mbasmanova two questions to clarify:
1. It's already next to ST_Distance in the .rst file; do you mean move the function implementation in .java next to ST_Distance?
2. For keeping the ST_Distance name, do you mean to just change _@ScalarFunction("ST_SphericalDistance")_ to _@ScalarFunction("ST_SphericalDistance")_ in the source file?

Reply:
@gaoxm 

Re: 1 - Sorry, I didn't realize they are next to each other already.

Re: 2 - Mostly, yes. `@ScalarFunction("ST_Distance")`